src/index.ts

```diff
@@ -6,8 +6,8 @@ import { onMessage } from './lib/Events'
 export class Plugin {
   public static client: CommandoClient
   public static config: Config
-  public static id = 'Plugin Template'
-  public static description = 'A template plugin'
+  public static id = 'Plugin Friends'
+  public static description = 'A friends plugin'
```

Comment:
Description should be a bit more descriptive. Use this: `Friend system plugin. Provides a better and more interactive friend system than Discord.`